app/views/helpers/errorSummary.scala.html

```diff
@@ -35,12 +37,13 @@ <h2 class="h2-heading error-summary-heading" id="errorMessageHeadingText">@Messa
                     }
                 ) { anchor =>
                 <li data-journey='@Messages("gmp.bulk.problem.error.field"):@Messages(formHeaderKey):@error.message'>
-                    <a href="#@{if(focusId.isDefined) focusId.get else anchor}" data-focuses="@{if(focusId.isDefined) focusId.get else error.key}">
+                    <a class="govuk-link" href="#@{if(focusId.isDefined) focusId.get else anchor}" data-focuses="@{if(focusId.isDefined) focusId.get else error.key}">
                         @Messages(error.message)
                     </a>
                 </li>
                 }
             }
-        </ul>
+            </ul>
+        </div>
     </div>
 }
```

Comment:
will this page still be needed after all the pages have been updated?  I thought that the new play UI integrated this into it so that a separate page was not needed.  If this is incorrect, it is okay to close/resolve conversation.

Reply:
This page won't be needed once all pages are turned to use new govuk component. This can't be deleted no as other pages might be using it. This should be a part of final cleanup ticket